### PR TITLE
feat: add audited contracts

### PR DESCRIPTION
**Pull request by mrice32**
This adds previously audited contracts to this repository.

Note: to move these over, I had to do a find/replace OVAL -> Oval. After that, the compilation seemed to work, but there may be other issues. If you notice something off, let me know and I can fix.

### CODE REVIEW

File: src/adapters/destination-adapters/ChronicleMedianDestinationAdapter.sol [hunk 1]
```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```
Comment (Reinis-FRP) [inline src/adapters/destination-adapters/ChronicleMedianDestinationAdapter.sol:1]: do we want mixing licenses? this repo currently has BUSL-1.1

File: src/adapters/destination-adapters/ChronicleMedianDestinationAdapter.sol [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+pragma solidity 0.8.17;
+
+import {IMedian} from "../../interfaces/chronicle/IMedian.sol";
+import {DiamondRootOval} from "../../DiamondRootOval.sol";
+
+/**
+ * @notice ChronicleMedianDestinationAdapter contract to expose Oval data via the standard Chronicle interface.
+ */
+
```
Comment (Reinis-FRP) [inline src/adapters/destination-adapters/ChronicleMedianDestinationAdapter.sol:10]: nit: `forge fmt` would remove empty line here and everywhere else after the contract NatSpec

**Review by Reinis-FRP — APPROVED** [review]

File: src/adapters/destination-adapters/ChronicleMedianDestinationAdapter.sol (hunk 1 above)
Comment (chrismaree) [reply src/adapters/destination-adapters/ChronicleMedianDestinationAdapter.sol:1]: @Reinis-FRP you're right. it should be the same license that this repo currently has.

**Review by chrismaree — APPROVED** [review]
LGTM! importantly, we are only adding new adapters & tests and all the base Oval logic goes un-touched.

**Pull request merged**